ambari-common/src/main/python/ambari_commons/repo_manager/__init__.py

```diff
@@ -61,6 +61,7 @@ def get_new_instance(cls, os_family=None):
 
     construct_rules = {
       OSConst.UBUNTU_FAMILY: AptManager,
+      OSConst.DEBIAN_FAMILY: AptManager,
```

Comment:
@hapylestat, I don't fully understand what you suggest.

The problem is that OSConst.UBUNTU_FAMILY is a simple string so it won't match to OSConst.DEBIAN_FAMILY.

```python
>>> type(os_check.OSConst.OS_UBUNTU)
<type 'str'>
```

However this one works correctly and returns True on Debian.

```python
>>> os_check.OSCheck.is_ubuntu_family()
True
```
So instead of the map lookup we could modify it to something like this:

```python
if OSCheck.is_ubuntu_family():
  return AptManager
if OSCheck.is_suse_family():
  return ZypperManager
if OSCheck.is_redhat_family():
  return YumManager
..
```